JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt

```diff
@@ -89,51 +88,47 @@ fun PostTitle(post: Post) {
 
 @Composable
 fun PostCardSimple(post: Post) {
-    Clickable(
-        modifier = Modifier.ripple(),
-        onClick = { navigateTo(Screen.Article(post.id)) }
+    Row(modifier = Modifier
+        .clickable(onClick = { navigateTo(Screen.Article(post.id)) })
```

Comment:
`clickable` should usually be after any `padding`s. This is so that the click region includes the padding.

Reply:
It's the other way around.